installation/cmd/run.sh

```diff
@@ -7,14 +7,16 @@ KYMA_RELEASE=${1:-$defaultRelease}
 COMPASS_HELM_RELEASE_NAME="compass"
 COMPASS_HELM_RELEASE_NAMESPACE="compass-system"
 
+CR_KYMA_LITE_PATH="${ROOT_PATH}"/installation/resources/installer-cr-kyma-diet.yaml
+
 kyma provision minikube
-kyma install -o "${ROOT_PATH}"/installation/resources/installer-cr-without-compass.yaml -o "${ROOT_PATH}"/installation/resources/installer-config.yaml --release "${KYMA_RELEASE}"
+kyma install -o $CR_KYMA_LITE_PATH --release "${KYMA_RELEASE}"
```

Comment:
Please change the variable name, maybe use something more generic like:

```suggestion
kyma install -o $INSTALLATION_CR_PATH --release "${KYMA_RELEASE}"
```